diff --git a/bin.cjs b/bin.cjs
index d0fd22888c12f76582c901a372534a29d33d6bb5..6db6ed231fdbf6d512f335f481412353914d5a65 100755
--- a/bin.cjs
+++ b/bin.cjs
@@ -17732,10 +17732,15 @@ var require_pgSerializer = __commonJS({
             const checkConstraints = {};
             const tableResponse = await getColumnsInfoQuery({ schema: tableSchema, table: tableName, db });
             const tableConstraints = await db.query(
-              `SELECT c.column_name, c.data_type, constraint_type, constraint_name, constraint_schema
+              `SELECT c.column_name, c.data_type, tc.constraint_type, tc.constraint_name, tc.constraint_schema
       FROM information_schema.table_constraints tc
-      JOIN information_schema.constraint_column_usage AS ccu USING (constraint_schema, constraint_name)
-      JOIN information_schema.columns AS c ON c.table_schema = tc.constraint_schema
-        AND tc.table_name = c.table_name AND ccu.column_name = c.column_name
-      WHERE tc.table_name = '${tableName}' and constraint_schema = '${tableSchema}';`
+      JOIN information_schema.key_column_usage AS kcu
+        ON kcu.constraint_schema = tc.constraint_schema
+        AND kcu.constraint_name = tc.constraint_name
+        AND kcu.table_schema = tc.table_schema
+        AND kcu.table_name = tc.table_name
+      JOIN information_schema.columns AS c ON c.table_schema = tc.table_schema
+        AND c.table_name = tc.table_name AND c.column_name = kcu.column_name
+      WHERE tc.table_name = '${tableName}' and tc.constraint_schema = '${tableSchema}'
+      ORDER BY tc.constraint_name, kcu.ordinal_position;`
             );
